supabase/functions/_backend/utils/ip.ts

```
@@ -0,0 +1,75 @@
+    return true
+  if (
+    normalized.startsWith('fe80:')
+    || normalized.startsWith('fc')
+    || normalized.startsWith('fd')
+    || normalized.startsWith('ff')
+    || normalized.startsWith('2001:db8:')
+    || normalized.startsWith('2001:2:')
+    || normalized.startsWith('2001:10:')
+    || normalized.startsWith('2002:')
+  )
+    return true
```

ESLint

[error] Expect curly brackets

(antfu/curly)